lib/WebpackConfig.js

```diff
@@ -57,16 +57,16 @@ class WebpackConfig {
         this.useSassLoader = false;
         this.useReact = false;
         this.usePreact = false;
-        this.preactOptions = {
-            preactCompat: false
-        };
         this.useVueLoader = false;
         this.useTypeScriptLoader = false;
         this.useForkedTypeScriptTypeChecking = false;
 
         // Features/Loaders options
         this.sassOptions = {
-            resolve_url_loader: true
+            resolveUrlLoader: true
+        };
+        this.preactOptions = {
```

Comment:
Not related to this PR but it wasn't at the right place since #152 was merged after #144